app/assets/javascripts/discourse/app/components/topic-timeline/container.gjs

```diff
@@ -659,6 +660,10 @@ export default class TopicTimelineScrollArea extends Component {
           />
         {{/if}}
 
+        {{#if (and this.currentUser this.site.desktopView)}}
```

Comment:
Oh I thought you removed them on purpose since they were already in the topic footer buttons. Maybe you only wanted to "hide" them when the topic footer buttons are shown instead?

Reply:
I'm honestly not sure anymore if I did it on purpose or not. I think it was accidental cause my redesign was scoped to mobile.